src/lib/timeseries.js

```diff
@@ -11,6 +11,7 @@ import sortBy from 'lodash/sortBy'
 import sumBy from 'lodash/sumBy'
 import toPairs from 'lodash/toPairs'
 import uniq from 'lodash/uniq'
+import { getPlaceLabelByContact } from 'src/components/ContactToPlace/helpers'
```

Comment:
This function should probably be in src/lib rather than in a component helper

Reply:
je prends ça pour un nit ;)